packages/app/src/systems/UI/components/Button.tsx

```diff
@@ -56,6 +56,17 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
     });
 
     const customProps = omit(["onPress"], props);
+    const loadingSpinnerSize = useMemo(() => {
```

Comment:
I think you can create a map outside of the function and just get property of it

```jsx
const SPINNER_SIZE = {
  sm: 16,
  md: 22,
  lg: 28
}

/* ... */

size={SPINNER_SIZE[props.size]}
```